skunkworks_crow/src/main/java/org/odk/share/views/ui/about/AboutActivity.java

```diff
@@ -55,11 +60,20 @@ protected void onCreate(Bundle savedInstanceState) {
     @Override
     public void onItemClick(View view, int position) {
         if (position == 0) {
+
+            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
```

Comment:
@devanshi7799 You can refer to ODK-Collect Codebase for Chrome Custom tab implementation, as much as I remember a lot of devices do not support CCT especially low-end devices. So we need to handle a lot of cases, and ODK-Collect code is also evolved with time and a lot of crashes are fixed after that feature addition, so we can take some inspiration from there and write the CustomTabs implementation for Skunkworks-crow also, but if you're referring it then do mention the filename you referred in the code.
For ref. https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/utilities/CustomTabHelper.java

Reply:
@lakshyagupta21 i have made some changes. please do review it.